### PR TITLE
Log hostname of the vm being migrated

### PR DESCRIPTION
**Pull request by mtkh**

### CODE REVIEW

**Comment by mtkh** [comment]
rested this please

**Comment by mtkh** [comment]
retest this please

File: igvm/kvm.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ def migrate_live(source, destination, vm, domain):
         (source.dataset_obj['os'], destination.dataset_obj['os'])
     )['flags']
 
-    log.info('Starting migration')
+    log.info('Starting migration of server: {}'.format(vm))
```
Comment (TuxPowered42) [inline igvm/kvm.py]: Why only only for live migration?

Comment (mtkh) [reply igvm/kvm.py]: good idea, its now logging for offline migration as well

**Pull request merged**